interpreter/cling/lib/Interpreter/IncrementalExecutor.cpp

```diff
@@ -171,8 +177,12 @@ IncrementalExecutor::NotifyLazyFunctionCreators(const std::string& mangled_name)
     if (ret)
       return ret;
   }
+  llvm::StringRef name(mangled_name);
+  void *address = nullptr;
+  if(m_externalIncrementalExecutor)
+   address = m_externalIncrementalExecutor->getAddressOfGlobal(name);
```

Comment:
This shouldn't happen here but in the IncrementalJIT which does the symbol lookup. Reason is that it might compile (codegen) a function which in turn needs a symbol - which should be resolved from the previous interpreter. So chaining the IncrementalJITs seems to be the right approach. They already have a name -> symbol function that we'll need to extend.